docker/Dockerfile

```diff
@@ -34,4 +33,6 @@ RUN /opt/conda/bin/pip install --no-cache-dir \
 RUN /opt/conda/bin/conda install -c conda-forge ffmpeg==4.2.3 && \
     /opt/conda/bin/conda clean --all
 
+RUN /opt/conda/bin/pip install --no-cache-dir adversarial-robustness-toolbox==1.4.2
+
```

Comment:
How does this work if it is moved to the pytorch/tf1/tf2 dockerfiles after the installation of those frameworks?